<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="radio2">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>150</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>message for each number</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>same msg for all number</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sendbutton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="msgarea">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>431</width>
      <height>133</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>171</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>chosse phone number file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="saveLocation">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>251</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="link">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>https://web.WhatsApp.com/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openlink">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>open link</string>
    </property>
   </widget>
   <widget class="QListWidget" name="phoneList">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>256</width>
      <height>341</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="movement">
     <enum>QListView::Static</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="uniformItemSizes">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>921</width>
      <height>421</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/HassanMahmoud/Desktop/e8d94e1e8b6b530ad315e9385290141b.jpg</pixmap>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>radio2</zorder>
   <zorder>radio1</zorder>
   <zorder>sendbutton</zorder>
   <zorder>msgarea</zorder>
   <zorder>browse</zorder>
   <zorder>saveLocation</zorder>
   <zorder>link</zorder>
   <zorder>openlink</zorder>
   <zorder>phoneList</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
